### PR TITLE
Adding Remi

### PR DESCRIPTION
**Pull request by Silun**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/Remi/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  host:
+    - pip
+    - python
+
+  run:
+    - python
+
+test:
+  imports:
+
```
Comment (npavlovikj) [inline recipes/Remi/meta.yaml]: You should test for `remi` here.

File: recipes/Remi/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  requires:
```
Comment (npavlovikj) [inline recipes/Remi/meta.yaml]: Remove `requires`, you don't need it.

File: recipes/Remi/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+
```
Comment (npavlovikj) [inline recipes/Remi/meta.yaml]: Please remove empty line.

File: recipes/Remi/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,43 @@
+{% set name = "remi" %}
+{% set version = "2020.1.16" %}
+{% set styledName = "Remi" %}
+
+
```
Comment (npavlovikj) [inline recipes/Remi/meta.yaml]: Please remove empty line.

File: recipes/Remi/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+  imports:
+
+  requires:
+
+about:
+  home: "https://github.com/dddomodossola/remi"
+  license: "Apache 2.0"
+  license_family: "Apache"
+  license_file: "license.txt"
+  summary: "Remi is a GUI library for Python applications that gets rendered in web browsers. This allows you to access your interface locally and remotely."
```
Comment (npavlovikj) [inline recipes/Remi/meta.yaml]: Please make the `summary` shorter, maybe just the first sentence. And you can use `description` for more details as you have now.

File: recipes/Remi/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,43 @@
+{% set name = "remi" %}
+{% set version = "2020.1.16" %}
+{% set styledName = "Remi" %}
```
Comment (npavlovikj) [inline recipes/Remi/meta.yaml]: Also, you don't use `styledName`anywhere, so you can just remove that line from the recipe.

**Comment by Silun** [comment]
Thank you, I made the changes you have listed.

**Comment by Silun** [comment]
AppVeyor failed once again. I don't see why the changes I made would let it fail, and I don't really understand what the problem is from the error it gives in the the details.

**Comment by npavlovikj** [comment]
@Silun , `noarch` recipes are only built on Linux, so we can safely ignore the AppVeyor failure in this case.

**Comment by npavlovikj** [comment]
Thank you for contributing and the prompt fixes @Silun 🎉

**Pull request merged**